prow/config.yaml

```diff
@@ -248,11 +248,37 @@ branch-protection:
         contexts:
         - cla/linuxfoundation
       repos:
+        apiserver-builder:
+          required_status_checks:
+            contexts:
+            - continuous-integration/travis-ci
         bootkube:
+          required_pull_request_reviews:
+            required_approving_review_count: 1
+          required_status_checks:
+            contexts:
+            - continuous-integration/travis-ci
+        cluster-proportional-autoscaler:
+          required_status_checks:
+            contexts:
+            - continuous-integration/travis-ci
+        external-storage:
           required_status_checks:
             contexts:
             - continuous-integration/travis-ci
         kube-arbitrator:
+          required_pull_request_reviews:
+            required_approving_review_count: 1
+            dismiss_stale_reviews: true
```

Comment:
These review requirements don't match the Tide configuration. The Tide query should include the following if this is to remain a merge requirement: https://github.com/kubernetes/test-infra/blob/a53fae5dffc30ebf962c89641159e06b9c1056bf/prow/config/tide.go#L146

The `review:approved` tag is described here: https://help.github.com/articles/searching-issues-and-pull-requests/#search-by-pull-request-review-status
The documentation is unclear about if this considers dismissed stale reviews.

Reply:
This sounds like a good check to add to the config_check!